nixos/modules/services/networking/fedimintd.nix

```diff
@@ -0,0 +1,363 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    filterAttrs
+    flatten
+    mapAttrs'
+    mapAttrsToList
+    mkDefault
+    mkEnableOption
+    mkIf
+    mkOption
+    mkPackageOption
+    nameValuePair
+    recursiveUpdate
+    types
+    ;
+
+  eachFedimintd = filterAttrs (fedimintdName: cfg: cfg.enable) config.services.fedimintd;
+  eachFedimintdNginx = filterAttrs (fedimintdName: cfg: cfg.nginx.enable) eachFedimintd;
```

Comment:
Not a blocker, but it would be great if you could move these down into a let block under config, for better code locality. Had to scroll up and down a few times to look at these. Something like this?

```nix
config = let
  eachFedimintd = filterAttrs (fedimintdName: cfg: cfg.enable) config.services.fedimintd;
  eachFedimintdNginx = filterAttrs (fedimintdName: cfg: cfg.nginx.enable) eachFedimintd;
in mkIf (eachFedimintd != { }) {
  ...
};
```